content/en/docs/Book/introduction.md

```diff
@@ -14,13 +14,21 @@ AWS, Google and Microsoft. We are asking ourselves if ethics have a place in
 our governance and licensing models. We are questioning the fundamental values
 and ideas of the movement itself.
 
-Since the term "Open Source" was first adopted in 1997, it has become the
-dominant software development model around the world. Back then, it was
-a cutting-edge idea - participating in open source was a radical act. Simply
-participating in it required a philosophical background. Today, you can
-actively participate in Open Source without any background at all in its
-philosophy. Open Source is in the water for the new generation of software
-developers.
+The term “Open Source” was first adopted in 1997. At the time, it was common
+practice for software to be distributed in binary/compiled form with
+restrictive licensing and no access to the source code. Meanwhile, for over a
+decade, liberal licenses had been used for software developed at universities
+for fostering collaboration. The underlying philosophy behind these licenses
+was to foster collaborative communities where a single institution can not
```

Comment:
Looks great! My one nitpick is that we say "for fostering collaboration" in one sentence, followed by "to foster collaboration in the next." With so many choices for words, we should be able to avoid the redundancy.

How about:

> Meanwhile, for over a decade, liberal licenses had been used for software developed at universities
*to encourage* collaboration. The underlying philosophy behind these licenses
was to foster collaborative communities where a single institution...